packages/web-config-server/src/apiV1/measureBuilders/DataPerOrgUnitBuilder.js

```diff
@@ -78,7 +78,10 @@ export class DataPerOrgUnitBuilder extends DataBuilder {
         throw new Error('The base data builder should return a single element array');
       }
 
-      return { organisationUnitCode, [dataElementCode]: data[0][dataElementCode] };
+      const value =
+        data[0][dataElementCode] !== undefined ? data[0][dataElementCode] : data[0].value; // key is sometimes changed to "value" by aggregator
```

Comment:
This is an optional change: I believe it makes the `dataElementCode` vs `'value'` logic a bit clearer, but feel free to ignore it if you think otherwise!

```suggestion
      const [dataItem] = data;
      // key is sometimes changed to "value" by aggregator
      const value = dataElementCode in dataItem ? dataItem[dataElementCode] : dataItem.value;
```

Also, the fact that aggregator sometimes uses `'value'` and sometimes `dataElementCode` is a smell for an underlying problematic logic. However, this is probably out of scope for this task: according to my experience, consistent single and multi-value data handling is a bit hard without significant refactoring of existing data/measure builders.

Reply:
Yeah I saw some comments saying why it's needed, happy to leave for someone else further down the road.

Good thinking with the simplifying this part, updated.